### PR TITLE
add time dependences to Potential and related functions

### PR DESCRIPTION
**Pull request by nstarman**
there are a lot of functions, like vcirc, that call time-dependent functions but do not themselves have a time parameter. I just went and found as many time-dependent functions as I could. in `galpy.potential` (and all functions that called those functions) and gave them time parameters.

### CODE REVIEW

File: galpy/potential/Potential.py [hunk 1]
```diff
@@ -578,7 +578,7 @@ def Rzderiv(self,R,Z,phi=0.,t=0.):
         except AttributeError: #pragma: no cover
             raise PotentialError("'_Rzderiv' function not implemented for this potential")      
 
-    def normalize(self,norm,t=0.):
+    def normalize(self,norm):
```
Comment (jobovy) [inline galpy/potential/Potential.py]: Why did you remove the time dependence here? I don't think it's used ever, that's true...

File: galpy/potential/plotEscapecurve.py [hunk 2]
```diff
@@ -145,14 +148,25 @@ def calcEscapecurve(Pot,Rs):
     except TypeError:
         grid=1
         Rs= nu.array([Rs])
+    try:
```
Comment (jobovy) [inline galpy/potential/plotEscapecurve.py]: I don't think it makes sense to allow an array for ``t`` here, because the point of the function is to compute the escape curve at a given time. So just change this to ``t`` being a number.

File: galpy/potential/plotEscapecurve.py [hunk 3]
```diff
@@ -120,7 +120,7 @@ def plotEscapecurve(Pot,*args,**kwargs):
     return plot.bovy_plot(Rs,esccurve,*args,
                           **kwargs)
 
-def calcEscapecurve(Pot,Rs):
+def calcEscapecurve(Pot,Rs,ts=0.):
```
Comment (jobovy) [inline galpy/potential/plotEscapecurve.py]: Docs say ``t``, for consistency with other methods, use ``t``.

File: galpy/potential/plotRotcurve.py [hunk 4]
```diff
@@ -158,14 +160,25 @@ def calcRotcurve(Pot,Rs,phi=None):
     except TypeError:
         grid=1
         Rs= nu.array([Rs])
+    try:
```
Comment (jobovy) [inline galpy/potential/plotRotcurve.py]: Like for the escape curve, only allow a scalar time here.

File: galpy/potential/plotRotcurve.py [hunk 5]
```diff
@@ -123,7 +123,7 @@ def plotRotcurve(Pot,*args,**kwargs):
     return plot.bovy_plot(Rs,rotcurve,*args,
                           **kwargs)
 
-def calcRotcurve(Pot,Rs,phi=None):
+def calcRotcurve(Pot,Rs,phi=None,ts=0.):
```
Comment (jobovy) [inline galpy/potential/plotRotcurve.py]: Should be ``t`` for consistency.

File: galpy/potential/Potential.py (hunk 1 above)
Comment (nstarman) [reply galpy/potential/Potential.py]: yeah, 1) because I could never find a time it's used and 2) because I couldn't think of a time when a potential would be normalized except at instantiation.

Comment (nstarman) [reply galpy/potential/Potential.py]: also, `normalize` wasn't passing the parameter to `Rforce`.
